config/_default/menus/main.en.yaml

```
@@ -4290,31 +4290,11 @@ menu:
       parent: dora_metrics
       identifier: dora_metrics_deployments
       weight: 2
-    - name: APM Deployment Tracking
-      url: dora_metrics/deployments/apm
-      parent: dora_metrics_deployments
-      identifier: dora_metrics_deployment_apm
-      weight: 201
-    - name: API
-      url: dora_metrics/deployments/deployment_api
-      parent: dora_metrics_deployments
-      identifier: dora_metrics_deployment_events_api
-      weight: 202
-    - name: Failure Data Sources
+    - name: Incident Data Sources
       url: dora_metrics/failures/
```

```suggestion
      url: dora_metrics/setup/failures/
```

I would move deployment and incident in the setup menu, so we clearly differentiate the "setup" from "learn more about the data" parts